app/components/dialogs/EditSongDialog.tsx

```
@@ -0,0 +1,228 @@
+        title="Release Date"
+        label="Release Date"
+        leadingIcon={null}
+        trailingIcon={null}
+      />
+
+      <div className="flex justify-end gap-2 mt-4">
+        <button
+          onClick={handleSave}
+          disabled={!songData.title || songData.artists.length === 0}
+          className="px-4 py-2 bg-[--md-sys-color-primary] text-[--md-sys-color-on-primary] rounded-md disabled:opacity-50"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)